content/en/docs/Reference/auroraboot.md

```diff
@@ -706,66 +706,37 @@ docker run -v "$PWD"/aurora.yaml:/aurora.yaml --rm -ti --net host quay.io/kairos
 ```
 
 
-## Booting with UKI and Secure Boot via EFI Key Enrollment
+## Booting with UKI and Secure Boot via HTTP Boot
 
-AuroraBoot now supports booting **Unified Kernel Images (UKI)** with **Secure Boot**. This process relies on enrolling platform keys at boot time via a helper binary (`efi-key-enroller`) and involves a two-stage PXE boot process. This feature is useful for environments enforcing Secure Boot, allowing boot and installation of Kairos from a signed UKI ISO.
+AuroraBoot now supports booting **Unified Kernel Images (UKI)** with **Secure Boot**. This process relies on using HTTP Boot to enroll the necessary keys and boot the UKI ISO directly as a UEFI 2.5 feature.
 
-> **Note**: Currently, only PXE boot has been tested and validated for this setup. HTTP boot support exists but has not been thoroughly tested.
+> **Note**: This is available on AuroraBoot v0.8.1 and later, and requires Kairos version 3.5.X or higher.
+> Currently, only HTTP boot has been tested and validated for this setup. PXE booting with UKI and Secure Boot is not yet supported and probably wont be.
```

Comment:
I think the notes would go well in an alert

Reply:
I was looking around to see if we had an especific alert for marking a feature available. Like a especific alert that marks the version a given feature is valuable from but I don't think so, I think we just use normal alerts for this.